src/main/java/com/ionosenterprise/rest/domain/BackupUnits.java

```diff
@@ -0,0 +1,16 @@
+package com.ionosenterprise.rest.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class BackupUnits extends BaseResource {
```

Comment:
javadoc

Reply:
I've followed existing code styles. If the javadoc is necessary a separate ticked will be created for this.